cpp/src/io/comp/io_uncomp.h

```diff
@@ -19,7 +19,7 @@
 #include <vector>
 #include <string>
 
-#include "cudf.h"
+#include <cudf/cudf.h>
```

Comment:
Since you're touching this include, might as well move this above the STL includes.